docs/README.md

```diff
@@ -245,5 +251,20 @@ To submit credentials, use Postman (or similar, based on your local configuratio
 ]
 ```
 
+## Aries VCR Client
```

Comment:
All of the "Aries-VCR" clients need to be containerized, using our standard `manage script approach.

Reply:
https://github.com/bcgov/orgbook-bc-client/pull/5

Reply:
I have also removed Dockerfiles for web specific builds as these have been moved into the client.

Please confirm whether any of these have been erroneously removed

https://github.com/bcgov/aries-vcr/pull/629/commits/a7d77164e64670e49862b2e6aa5f3ffc1d96f894

Reply:
That is the correct approach.  This is just a comment to indicate the clients that have been split out need to be containerized and support our typical `manage` script workflows to make them easy for people to use with `aries-vcr`.